.github/workflows/Analyze.yml

```diff
@@ -0,0 +1,39 @@
+name: Analyze
+
+on:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'java' ]
+
+    steps:
+    - uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+
+    - uses: ./.github/actions/setup-java
+
+    - name: Initialize CodeQL
+      uses: github/codeql-action/init@v2
+      with:
+        languages: ${{ matrix.language }}
+
+    - name: Build project
+      uses: gradle/gradle-build-action@243af859f8ca30903d9d7f7936897ca0358ba691 # v2
+      with:
+        gradle-home-cache-cleanup: true
+        arguments: assemble --stacktrace
```

Comment:
I think assemble task spends too much time
how about do assemble task only :app-android module?

Reply:
I see! Regarding the build, I was referring to the Build workflow 👀